### ISSUE TITLE
WintunOpenAdapter: Wintun adapter with the same name exists in another pool

### ISSUE THREAD

**Issue by Xe** [tailscale/tailscale#1128]
**Describe the bug**
Tailscale fails to launch on Windows 10 pro, coincidentally after renaming the PC. Logtail says the following:

```
13 Jan 21 20:03 -0800: logid: f6ec2cbeea211e7a2d404669d002e377ddc834594ec98cbc610d239a664aab40
13 Jan 21 20:04 -0800: ipnserver: try2: trying getEngine again...
13 Jan 21 20:04 -0800: Starting userspace wireguard engine with tun device "Tailscale"
13 Jan 21 20:04 -0800: [Wintun] WintunOpenAdapter: Wintun adapter with the same name exists in another pool
13 Jan 21 20:04 -0800: [Wintun] CreateAdapter: Creating adapter
13 Jan 21 20:04 -0800: [Wintun] CreateAdapter: Failed to setup adapter: Cannot create a file when that file already exists. (Code 0x000000B7)
13 Jan 21 20:04 -0800: no TUN failure diagnostics for OS "windows"
13 Jan 21 20:04 -0800: CreateTUN: Error creating interface: Cannot create a file when that file already exists.
13 Jan 21 20:04 -0800: ipnserver2: getEngine failed again: wgengine.NewUserspaceEngine: Error creating interface: Cannot create a file when that file already exists.
```

**To Reproduce**
Reproduction steps unknown at this time.

**Expected behavior**
Tailscale launches.

**Screenshots**
![image](https://user-images.githubusercontent.com/529003/104531683-b0a98d80-55dc-11eb-84c4-f94a1ed8c265.png)

**Version information:**
 - Device: KVM virtual machine
 - OS: Windows
 - OS version: Windows 10
 - Tailscale version: 1.3.147.0

**Additional context** 
There are similar errors in https://github.com/tailscale/tailscale/issues/1055, the Tailscale network driver is left in Windows 10 but I cannot do anything to control it from Control Panel.

**Comment by Xe** [comment]
I was able to work around this by deleting the Wintun driver from device manager and relaunching Tailscale.

**Comment by bradfitz** [comment]
Is the machine renaming key here? Were you able to reproduce by renaming the machine again?

/cc @peske

**Comment by peske** [comment]
wintun.dll first checks if the driver exists, and then installs it if it doesn't. It looks that in this case (probably because machine rename), wintun.dll fails to find the driver and tries to install it, but then Windows complains that the driver already exists. I'll try to reproduce, and if there is a problem, it needs to be fixed in wintun.dll itself.

**Comment by zx2c4** [comment]
https://git.zx2c4.com/wintun/commit/?id=49b7692d5becd86d4ff4854ed089ddd4c3d93b15 This should fix it and will be available in Wintun 0.10.1 when that's released.

**Comment by bradfitz** [comment]
@zx2c4, thanks!

**Comment by bradfitz** [comment]
Wintun 0.10.1 is out. This'll be released in the next Tailscale 1.4.x build for Windows (anything greater than 1.4.2)

**Comment by JohnDDuncanIII** [comment]
Hello! I just ran into this same issue earlier this evening.

Tailscale was working until I renamed my computer from its auto-generated name in the https://login.tailscale.com/admin/machines web interface.

My tailscale ip for this machine is `100.93.181.66`.

I'm running `Tailscale 1.8.7` on the latest 64bit release of Windows 10.

```
Edition	Windows 10 Pro
Version	21H1
Installed on	‎10/‎27/‎2020
OS build	19043.1055
Experience	Windows Feature Experience Pack 120.2212.2020.0
```

---

![tailscale](https://user-images.githubusercontent.com/9257785/122498731-a3466400-cfbd-11eb-8bbf-7b7804e9c101.PNG)

![Capture](https://user-images.githubusercontent.com/9257785/122498734-a4779100-cfbd-11eb-8c6d-1e6756ded46f.PNG)

---

Following @Xe's [advice](https://github.com/tailscale/tailscale/issues/1128#issuecomment-759868046), I was able to fix this by 

- manually uninstalling the Wintun driver from the Windows device manager
- reinstalling Tailscale
- rebooting

I assume the reinstall/reboot steps were probably unnecessary given the conversation above.

One weird thing I noticed is that, after reinstalling and re-running Tailscale, the wintun driver no longer shows up in the Device Manager even though I assume the Tailscale service reinstalled it automatically.

![Devices](https://user-images.githubusercontent.com/9257785/122499324-b9085900-cfbe-11eb-91be-c67484a01717.PNG)

![Devices1](https://user-images.githubusercontent.com/9257785/122499325-ba398600-cfbe-11eb-8f44-990faf532030.PNG)
